api_tests/lib/bitcoin.ts

```diff
@@ -0,0 +1,165 @@
+import {
+    Transaction,
+    ECPair,
+    Out,
+    payments,
+    networks,
+    TransactionBuilder,
+    address,
+} from "bitcoinjs-lib";
+const BitcoinRpcClient = require("bitcoin-core");
+const sb = require("satoshi-bitcoin");
+const util = require("./util.js");
+
+// TODO: Remove regtest
+// Once this is merged: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33571
+
+const networksAny: any = networks;
+const regtest: any = networksAny.regtest;
+
+interface IBitcoinConfig {
+    // snake_case because it comes from TOML file
+    rpc_username: string;
+    rpc_password: string;
+    rpc_host: string;
+    rpc_port: number;
+}
+
+interface IBitcoinRpcClient {
+    // TODO: Create Interface for Promise returned by RPC calls
+    // We should avoid to use `any` and instead create the interface
+    // of what is returned by the RPC calls
+    generate(num: number): Promise<any>;
+    getRawTransaction(
+        txId: string,
+        verbose?: boolean,
+        blockHash?: string
+    ): Promise<any>;
+    sendToAddress(address: string, amount: number | string): Promise<any>;
+    sendRawTransaction(hexString: string): Promise<any>;
+}
+
+interface IUtxo {
+    // TODO: declare transactionId type
```

Comment:
❓  The bitcoin typescript didn't have a type for this?

Reply:
They only have that:
```
export interface Out {
    script: Buffer;
    value: number;
}
```